errors.js

```diff
@@ -9,6 +9,10 @@ exports.customErrors = (err, req, res, next) => {
 
 exports.psqlErrors = (err, req, res, next) => {
   if (err.code === '22P02') {
+    // 22P02: Invalid input type/syntax
+    res.status(400).send({ msg: 'Bad Request' });
+  } else if (err.code === '42703') {
+    // 42703: Column does not exist
     res.status(400).send({ msg: 'Bad Request' });
```

Comment:
As your resulting behaviour is identical at this conditional statement, consider grouping together

Reply:
Good point!